<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterForm</class>
 <widget class="QWidget" name="FilterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Фільтрувати за</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>441</width>
     <height>531</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="name_gridLayout">
      <item row="1" column="1">
       <widget class="QComboBox" name="name_filter_type_comboBox"/>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="name_value_le"/>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="project_name_checkBox">
        <property name="text">
         <string>Назвою проекту</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="manager_gridLayout">
      <item row="1" column="0">
       <widget class="QLineEdit" name="manager_value_le"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="manager_filter_type_comboBox"/>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="manager_checkBox">
        <property name="text">
         <string>Відповідальною особою</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="0" column="2">
       <widget class="QDateEdit" name="start_date"/>
      </item>
      <item row="1" column="2">
       <widget class="QDateEdit" name="end_date"/>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="date_checkBox">
        <property name="text">
         <string>Датою                            </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Від</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>До</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="time_gridLayout">
      <item row="1" column="0">
       <widget class="QLineEdit" name="time_value_le"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="time_filter_type_comboBox"/>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="time_checkBox">
        <property name="text">
         <string>Запланованим часом</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="status_gridLayout">
      <item row="1" column="0">
       <widget class="QLineEdit" name="status_value_le"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="status_filter_type_comboBox"/>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="status_checkBox">
        <property name="text">
         <string>Статусом</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="filter_pb">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>590</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Фільтрувати</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_filters_pb">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>590</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Скинути фільтри</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
